packages/@lwc/perf-benchmarks/package.json

```diff
@@ -6,7 +6,7 @@
         "build": "rm -fr dist && rollup -c  ./rollup.config.mjs && node scripts/build.js && ./scripts/fix-deps.sh",
         "test": "yarn test:run && yarn test:format",
         "test:run": "for file in $(find dist -name '*.tachometer.json'); do tach --config $file --json-file $(echo $file | sed 's/.json/.results.json/'); done",
-        "test:format": "node ./scripts/format-results.mjs $(find dist -name '*.results.json')",
+        "test:format": "if [ -z $BENCHMARK_SMOKE_TEST ]; then node ./scripts/format-results.mjs $(find dist -name '*.results.json'); fi",
```

Comment:
For the smoke test, formatting the results doesn't work because there is no comparison result for `master`